src/libs/SearchUtils.ts

```diff
@@ -674,6 +675,15 @@ function isCannedSearchQuery(queryJSON: SearchQueryJSON) {
     return !queryJSON.filters;
 }
 
+/**
+ * New Search Router is under construction and for now should be displayed only in dev.
+ *
+ * After everything is implemented this function can be removed, as we will start to always use SearchRouter.
+ */
+function shouldDisplayNewSearchRouter() {
+    return Environment.isDevelopment();
+}
```

Comment:
Maybe we should move this to `Permissions.ts`, since that's where we usually put stuff like this.

Reply:
Sounds good, didn't know about this file